File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/workbook/MergeWorkbooks.java
package com.aspose.cells.cloud.examples.workbook;

import com.aspose.cells.cloud.examples.Utils;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class MergeWorkbooks {

    public static void main(String... args) throws IOException {
        String input = "Sample1.xlsx";
        String output = "Sample2.xlsx";
        Path inputFile = Utils.getPath(MergeWorkbooks.class, input);
        Path outputFile = Utils.getPath(MergeWorkbooks.class, output);

        Utils.getStorageSdk().PutCreate(
                input,
                null,
                Utils.STORAGE,
                inputFile.toFile()
        );
        
        Utils.getStorageSdk().PutCreate(
                output,
                null,
                Utils.STORAGE,
                outputFile.toFile()
        );

        Utils.getCellsSdk().PostWorkbooksMerge(
                input, 
                output, 
                null, 
                null
        );
                

        com.aspose.storage.model.ResponseMessage sr
                = Utils.getStorageSdk().GetDownload(
                        input,
                        null,
                        Utils.STORAGE
                );

        Files.copy(sr.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);

    }
}

File: Examples/Node.js/SDK/workbook/MergeWorkbooks.js
var fs = require('fs'),
    assert = require('assert'),
    utils = require('../utils.js'),
    cells = utils.cellsApi,
    storage = utils.storageApi;



var inputFile1 = "sample1.xlsx";
var inputPath1 = utils.getPath(__filename, inputFile1);

var inputFile2 = "sample2.xlsx";
var inputPath2 = utils.getPath(__filename, inputFile2);

var outputName = "output.xlsx";
var outputPath = utils.getPath(__filename, outputName);


storage.PutCreate(inputFile1, null, null, inputPath1, function (responseMessage) {

   storage.PutCreate(inputFile2, null, null, inputPath2, function (responseMessage) {

        cells.PostWorkbooksMerge(inputFile2, inputFile1, null, null, function (responseMessage) {

            assert.equal(responseMessage.status, 'OK');
           // var destFileName = responseMessage.body.Workbook.FileName;
           // console.log("FileName " + destFileName);

            storage.GetDownload(inputFile1, null, null, function (responseMessage) {
                fs.writeFileSync(outputPath, responseMessage.body);
            });
        });
    });
});